### DISCUSSION TITLE
Outreach Applicant

### DISCUSSION THREAD

**Discussion by richarddushime** [LesFruitsDefendus/saskatoon-ng#160] [Outreachy]
Greetings to you All 
Hope you are doing Well 
i am Dushime Mudahera Richard an outreach applicant  And  loved this project , looking foward to contribute  and learn from each other 
thank you

**Comment by richarddushime** [comment]
Hello Mentors ,
@tristanlatr 
i worked on my project proposal  [Proposed Solution](https://docs.google.com/document/d/1SZyGiGxolXzim680Et0a1pdWxwQJJ7Gk4IF58orIFfU/edit?usp=sharing)

i am asking if you can helpme out where i correct

**Reply by tristanlatr** [reply]
Hello @richarddushime, sorry for the later reply, submit your final application with you proposed schedule if it’s not already done. 

I cannot take the time to review all schedules for everyone individually and send feedback, unfortunately.
